docs/pathways/semantic-engineer.md

```diff
@@ -0,0 +1,49 @@
+# Semantic Engineer Pathway
+
+## Getting Set-up
+
+1. Download and install [GitHub Desktop](https://desktop.github.com/)
+2. Download and install [Protege](https://protege.stanford.edu/products.php)
+3. Install [ELK reasoner in protege](../howto/installing-elk-in-protege.md)
+4. [Setting up ODK](../howto/odk-setup.md)
+
+## Learning
+
+As a semantic engineer, it would be useful for you to know how curators work, as such, it would be useful to be familiar with all the concepts in the [ontology curator pathways document](ontology-curator.md). This pathways will however be focusing on the engineering side of things.
+
+### Very basics
+
+1. [Basic introduction to CLI 1](../tutorial/intro-cli-1.md)
+1. [Basic introduction to CLI 2](../tutorial/intro-cli-2.md)
+
+### Learning Git and GitHub
+
+1. [Fundamentals of GitHub](../tutorial/github-fundamentals.md)
+1. [Github issues](../tutorial/github-issues.md)
+1. [Cloning a Repo](../howto/clone-mondo-repo.md)
+1. [Creating pull requests](../howto/github-create-pull-request.md)
+
+### Learning semantic engineering
+
+1. [ROBOT tutorial pt. 1](../tutorial/robot-tutorial-1.md)
+1. [ROBOT tutorial pt. 2](../tutorial/robot-tutorial-2.md)
+1. [Templates](../lesson/templates-for-obo.md)
+1. [Getting started with DOSDP templates](../tutorial/dosdp-overview.md)
+1. [DOSDP Templates Basic Tutorial](../tutorial/dosdp-template.md)
+1. [Introduction to Managing DOSDP Templates in ODK](../tutorial/dosdp-odk.md)
+1. [Basics of SPARQL](../tutorial/sparql.md)
+1. OAK
```

Comment:
@matentzn I probably got to the most important value-add right at the end, and didn't have much time to think/write about it. I added these 5 bullet points to the bottom of "learning semantic engineering", but I did not yet create pages for these.